CHANGELOG.md

```diff
@@ -83,7 +105,6 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ### Added
 
-- **HYRAX-439** - Corrected README.md according to a new Hyrax URL format.
```

Comment:
Is removing this intentional? Seems like we don't want to remove this

Reply:
40bd1b6e7